tests/Unit/Css/CssDocumentTest.php

```diff
@@ -488,6 +525,32 @@ public function notRendersInvalidNonConditionalAtRule(string $atRuleCss, string
         self::assertStringNotContainsString($atAndRuleName, $result);
     }
 
+    /**
+     * @test
+     *
+     * @param string $atRuleCss
+     * @param string $cssBefore
```

Comment:
We can drop these two as these are redundant with the native type annotations.

Reply:
Removed.